doc/pr2_tutorials/planning/launch/move_group_interface_tutorial.launch

```diff
@@ -1,7 +1,5 @@
 <launch>
 
-  <include file="$(find pr2_moveit_config)/launch/demo.launch"/>
-
```

Comment:
Note the python version of this tutorial now also uses two launch commands: https://github.com/ros-planning/moveit_tutorials/pull/47